flink-table/flink-table-planner/pom.xml

```diff
@@ -136,17 +136,17 @@ under the License.
 			<groupId>org.apache.calcite</groupId>
 			<artifactId>calcite-core</artifactId>
 			<!-- When updating the Calcite version, make sure to update the dependency exclusions -->
-			<version>1.18.0</version>
+			<version>1.19.0</version>
 			<exclusions>
 				<!--
 
 				Dependencies that are not needed for how we use Calcite right now.
 
-				"mvn dependency:tree" as of Calcite 1.18:
+				"mvn dependency:tree" as of Calcite 1.19:
 
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.18.0:compile
+				[INFO] +- org.apache.calcite:calcite-core:jar:1.19.0:compile
 				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.13.0:compile
```

Comment:
`calcite-avatica` 1.15.0 has released recently, do we have plan to upgrade ?

Reply:
I think we will use whatever version `calcite-core` is using. thus avatica 1.15.0 will be upgraded when upgrade to calcite 1.20.0